### DISCUSSION TITLE
How to change color of PasswordToggleIcon in PasswordInput with styles api in provider?

### DISCUSSION THREAD

**Discussion by 7iomka** [mantinedev/mantine#2158] [Q&A] [answered]
How to change color of PasswordToggleIcon in PasswordInput with styles api in provider?
I changed my palette for gray colors and now I have unreadable password toggle icon and I want to change somehow it color, but code below does nothing, even if docs say that I can use `icon` prop for styles

```ts
PasswordInput: {
      styles: (theme) => ({
        innerInput: {
          paddingLeft: 'inherit',
          height: 'inherit',
          paddingTop: 'inherit',
        },
        icon: {
          color: theme.colors.gray[9], // does nothing
        },
      }),
    },

```

**Comment by rtivital** [comment, marked as answer]
Check component styles api tab

visibilityToggle | .mantine-PasswordInput-visibilityToggle | Visibility toggle button
-- | -- | --
